analyzers/tests/SonarAnalyzer.UnitTest/Rules/Utilities/CopyPasteTokenAnalyzerTest.cs

```diff
@@ -104,21 +101,15 @@ public void Verify_Duplicated_CS_GlobalUsings()
             });
 
         [TestMethod]
-        public void Verify_NotRunForTestProject_CS()
-        {
-            const string testRoot = BasePath + nameof(Verify_NotRunForTestProject_CS);
-            CreateBuilder(new TestCopyPasteTokenAnalyzer_CS(testRoot, true), "DuplicatedDifferentLiterals.cs", AnalyzerLanguage.CSharp, testRoot)
-                .VerifyUtilityAnalyzerProducesEmptyProtobuf();
-        }
+        public void Verify_NotRunForTestProject_CS() =>
+            CreateBuilder(ProjectType.Test, "DuplicatedDifferentLiterals.cs").VerifyUtilityAnalyzerProducesEmptyProtobuf();
 
         [DataTestMethod]
         [DataRow("Unique.cs", true)]
         [DataRow("SomethingElse.cs", false)]
         public void Verify_UnchangedFiles(string unchangedFileName, bool expectedProtobufIsEmpty)
         {
-            var testRoot = BasePath + nameof(Verify_UnchangedFiles);
-            var builder = CreateBuilder(new TestCopyPasteTokenAnalyzer_CS(testRoot, false), "Unique.cs", AnalyzerLanguage.CSharp, testRoot)
-                .WithSonarProjectConfigPath(TestHelper.CreateSonarProjectConfig(TestContext, new[] { BasePath + unchangedFileName }));
+            var builder = CreateBuilder(ProjectType.Product, "Unique.cs").WithSonarProjectConfigPath(TestHelper.CreateSonarProjectConfig(TestContext, new[] { BasePath + unchangedFileName }));
```

Comment:
`CreateSonarProjectConfig` is called in a lot of places with a single file name. We should create an overload that accepts a string, creates the array, and calls the other overload.

Reply:
I had to rename it to a more explicit version because there's already another `CreateSonarProjectConfig` functionality with these arguments (`CreateSonarProjectConfigWithFilesToAnalyze`)